storage/client/src/main/java/com/azure/storage/blob/BlockBlobClient.java

```diff
@@ -154,7 +188,7 @@ public void uploadFromFile(String filePath) throws IOException {
 
     public void uploadFromFile(String filePath, BlobHTTPHeaders headers, Metadata metadata,
             BlobAccessConditions accessConditions, Duration timeout) throws IOException {
-        Mono<Void> upload = this.blockBlobAsyncClient.uploadFromFile(filePath, headers, metadata, accessConditions, null);
+        Mono<Void> upload = this.blockBlobAsyncClient.uploadFromFile(filePath, headers, metadata, accessConditions);
```

Comment:
So you removed the Context parameter from the async method, and so removed it from this call. But then you added a blockSize integer parameter to the method, and didn't add it to this call. This doesn't compile.

Reply:
Latest commit has this fixed.